airflow/docker.go

```diff
@@ -7,21 +7,24 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"strconv"
 	"strings"
 	"text/tabwriter"
+	"time"
 
 	"github.com/astronomer/astro-cli/config"
 	"github.com/astronomer/astro-cli/messages"
 
+	"github.com/compose-spec/compose-go/loader"
+	cTypes "github.com/compose-spec/compose-go/types"
```

Comment:
is it default naming convention? cType confuse me a bit.
wdyt to rename to more clear name `composeTypes` or `compTypes`?

Reply:
`composeTypes` make more sense, will update it 👍